src/Appwrite/Utopia/Response/Model/ProviderRepository.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class ProviderRepository extends Model
```

Comment:
Why don't we just call it repository ? `ProviderRepository` sounds forced.

Reply:
Provider prefix for everythinf external. `Reoutory` would probably mean Appwrit eRepository document, which is documented console DB that links function to a git repository ID